tests/test_inner.py

```diff
@@ -1411,3 +1412,54 @@ def test_fs_watcher(monkeypatch):
     w.join(0.1)  # timeout if thread still running
     assert not w.is_alive()
     assert "mb_used" in w.get_usage_data()
+
+
+@pytest.mark.parametrize(('yml_config', 'expected'), [
+    (dedent("""\
+     set_release_env: TEST_RELEASE_VAR
+     """),
+     ['FROM scratch\n', 'ENV TEST_RELEASE_VAR=1\n']),
+    (dedent("""\
+     """),
+     ['FROM scratch\n']),
+])
+def test_set_release_var(yml_config, expected, tmpdir):
+    class FakeReleaseBuilder(object):
+        def __init__(self, df_path):
+            self.df_path = df_path
+
+    tmpdir_str = str(tmpdir)
+
+    fake_image_uri = os.path.join(tmpdir_str, 'fake_imag_uri')
+    os.mkdir(fake_image_uri)
+    if os.path.exists('fake_imag_uri'):
+        os.removedirs('fake_imag_uri')
+    os.mkdir('fake_imag_uri')
+
+    yaml_path = os.path.join(fake_image_uri, REPO_CONTAINER_CONFIG)
+    with open(yaml_path, 'w') as yf:
+        yf.write(yml_config)
+        yf.flush()
```

Comment:
I don't think flush should be needed here as we're closing the file as soon as we leave the 'with' context.